src/components/admin/services/judging/Judging.jsx

```diff
@@ -22,6 +22,7 @@ const Judging = () => {
         judgesView={judges}
         setJudgesView={setJudges}
       />
+      {view ? <Title title="Judges" /> : <Title title="Teams" />}
```

Comment:
instead of having 2 components u can simplify down to changing the string in the component
```suggestion
      <Title title={view ? "Judges" : "Teams"} />
```